compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -334,6 +333,15 @@ class t_py_generator : public t_generator {
 
   std::string package_dir_;
   std::string module_;
+
+protected:
+  virtual std::set<std::string> lang_keywords() {
+    std::string keywords[] = { "False", "None", "True", "and", "as", "assert", "break", "class",
+          "continue", "def", "del", "elif", "else", "except", "exec", "finally", "for", "from",
+          "global", "if", "import", "in", "is", "lambda", "nonlocal", "not", "or", "pass", "print",
```

Comment:
Are "exec" and "print" python keywords?

Reply:
"exec" and "print" are python2 keywords. I merged python2 and python3 keyword lists.

Reply:
> It will be fun to add more keyword lists over time, for example for perl.

You have a strange understanding of fun …